Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/admingui/dashboard/DashboardStatusWebservice.java

```diff
@@ -79,41 +85,76 @@ public DashboardStatusMessageType dashboardStatus(AdminGUIRequestMessageType req
     }
 
     @Override
-    public LogEventSimpleResponseMessageType getSearchFilter(GetSearchFilterRequestMessageType arg0) {
+    public LogEventSimpleResponseMessageType getSearchFilter(GetSearchFilterRequestMessageType request) {
```

Comment:
Question: I don't see that we use request variable here. If not, why do we have it?

Reply:
the method is generated by the wsdl do we wanted to removed the request message?